tests/security/selinux/selinux_setup.pm

```diff
@@ -33,14 +33,21 @@ sub run {
         assert_script_run('zypper -n in policycoreutils-python');
     }
 
-    my $pkgs = script_output("echo `zypper se selinux | grep -i selinux | grep -v -e srcpackage -e debuginfo -e debugsource -e busybox | cut -d '|' -f 2`");
+    # install as many as SELinux related packages
+    my $pkgs
+      = "selinux-tools libselinux-devel libselinux1 libselinux1-32bit python3-selinux restorecond mcstrans libsepol1 libsepol-devel libsemanage1 libsemanage-devel checkpolicy setools-console";
```

Comment:
I wonder if we should have such an extesnive list - or if we should rather see if enabling SELinux like a user would do it works.

Things like libselinux1 must be part of the dependency chain after all

Reply:
Thanks for your comments.
The "selinux_setup.pm" is a preparation for "selinux_smoke.pm" and other following test cases.
The extensive list is not so much necessary (at least no harm) but sometime installing some pkgs may make something broken at the beginning of introducing SELinux. So we keep this testing strategy.